src/navigation-bar.tsx

```diff
@@ -129,11 +242,1008 @@ type MainNavigationBarPropsBase = {
+                onMouseLeave={() => cancelDebouncedOpenSectionMenu(index)}
+                onFocus={() => {
+                    if (menu && !hasCustomInteraction) setIsMenuControlFocused(true);
+                }}
+                onBlur={() => {
+                    if (menu && !hasCustomInteraction) setIsMenuControlFocused(false);
+                }}
+            >
+                <BaseTouchable
+                    {...getSectionInteractiveProps(touchableProps as InteractiveProps)}
+                    aria-label={
+                        !hasCustomInteraction
+                            ? `${section.title}, ${texts.mainNavigationBarOpenSectionMenu || t(tokens.mainNavigationBarOpenSectionMenu)}`
+                            : undefined
+                    }
+                    aria-haspopup={!hasCustomInteraction}
+                    aria-expanded={!hasCustomInteraction ? isSectionMenuOpen : undefined}
+                    className={classnames(
+                        styles.section,
+                        {
+                            [styles.selectedSectionVariantes[variant === 'inverse' ? 'inverse' : 'default']]:
+                                index === selectedIndex,
+                        },
+                        styles.textWrapperVariants[variant === 'inverse' ? 'inverse' : 'default']
+                    )}
+                >
+                    <Text3 regular color="inherit">
+                        {title}
+                    </Text3>
+                </BaseTouchable>
+            </div>
+            {menu && (
+                <>
+                    {hasCustomInteraction && (
+                        <div
+                            className={styles.desktopMenuSectionArrowContainer}
+                            onFocus={() => setIsMenuControlFocused(true)}
+                            onBlur={() => setIsMenuControlFocused(false)}
+                        >
+                            <BaseTouchable
+                                className={styles.desktopMenuSectionArrow}
+                                aria-label={`${section.title}, ${texts.mainNavigationBarOpenSectionMenu || t(tokens.mainNavigationBarOpenSectionMenu)}`}
+                                aria-haspopup
+                                aria-expanded={isSectionMenuOpen}
+                                onPress={() => {
+                                    if (isMenuControlFocused) {
+                                        menuButtonOnPress();
+                                    }
+                                }}
+                                style={{
+                                    pointerEvents: isMenuControlFocused ? 'auto' : 'none',
+                                    opacity: isMenuControlFocused ? 1 : 0,
+                                }}
+                            >
+                                <IconChevronLeftRegular
+                                    size={8}
```

Comment:
```suggestion
                                    size={12}
```

In specs the icon is 8px but all Mística Icons have a security area around the icon, so the real size is 12

Reply:
Discussed offline with Hector and updated the chevron